### PR TITLE
[readme] Update with new messaging and illustrations

### PR DESCRIPTION
**Pull request by kmelve**
Do not delete the instructional comments. -->

**Type of change (check at least one)**

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x]  Documentation (fix or update to documentation)
- [ ]  Maintenance
- [ ]  Other, please describe:

**Does this change require a documentation update? (Check one)**

- [ ]  Yes
- [x]  No

**Current behavior**

The readme was a bit stale and outdated.

**Description**

I have lifted the getting started instructions to the top. Summarized key features with helpful links. Updated illustrations. Removed a lot of the links that can be found on our site anyways.

**Note for release**

Updated the readme for the main repository.

**Checklist** 

- [x]  I have read the [Contributing Guidelines](https://github.com/sanity-io/sanity/blob/next/CONTRIBUTING.md)
- [ ]  The PR title includes a link to the relevant issue
- [x]  The PR title is appropriately formatted: `[some-package] PR title (#123)`
- [x]  The code is linted
- [x]  The test suite is passing
- [x]  Corresponding changes to the documentation have been made

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,166 +1,88 @@
...
-  <a href="https://slack.sanity.io/"><img src="https://slack.sanity.io/badge.svg"></a>
-</p>
+![npm](https://img.shields.io/npm/v/@sanity/base) ![Downloads](https://img.shields.io/npm/dm/@sanity/client) [![](https://slack.sanity.io/badge.svg)](https://slack.sanity.io/) ![Follow on twitter](https://img.shields.io/twitter/follow/sanity_io?style=flat&color=blue)
 
-## Table of contents
-
-- [Table of contents](#table-of-contents)
-- [Getting started](#getting-started)
-- [Key Features](#key-features)
-- [Plugins and tooling](#plugins-and-tooling)
```
Comment (evenwestvang) [inline README.md]: Don't we want to list plugins directly from the  Github?

File: README.md [hunk 2]
```diff
@@ -1,166 +1,88 @@
...
-  <a href="https://www.sanity.io">
-    <img width="40%" alt="Sanity Logo" src="https://www.sanity.io/static/images/logo_red.svg?v=2"/>
-  </a>
-</h1>
+# ![Sanity logo](https://cdn.sanity.io/images/3do82whm/next/51af00784c5addcf63ae7f0c416756acca7e63ac-353x71.svg?w=180&fm=png&fit=max) – Build with Structured Content
 
-<h4>
-  <a href="https://www.sanity.io">Sanity</a> is a real-time content infrastructure. The editor, built in JavaScript and React.js, lives in this repo. It connects to a scalable, hosted backend featuring a Graph Oriented Query Language (GROQ), asset pipelines and fast edge caches.
-</h4>
+> **Sanity.io is the platform for structured content, that comes with an open-souce editing environment called Sanity Studio that you can customize with JavaScript, and a hosted real-time data store.**
```
Comment (evenwestvang) [inline README.md]: Too long? How about:

> Sanity.io is the platform for structured content. It comes with an open-source editing environment called Sanity Studio that you can customize with JavaScript and a real-time hosted data store. Sanity has generous included quotas – so [getting started](https://www.sanity.io/create) is free.

File: README.md (hunk 1 above)
Comment (kmelve) [reply README.md]: I don't think we should try having all of them, so a selection perhaps?

File: README.md [hunk 3]
```diff
@@ -1,166 +1,88 @@
...
 
-- [Vision - GROQ sandbox tool](https://www.npmjs.com/package/@sanity/vision) (`sanity install @sanity/vision`)
+### [Structured Content](https://www.sanity.io/structured-content)
 
-### Schema plugins
+- Distribute from a single source of thruth, in real-time.
+- Unlock programmatic approaches to workflows and design.
+- [Upload images and transform them on demand](https://www.sanity.io/docs/asset-pipeline).
+- Query images for metadata like dominant colors, sizes, geo-location, and EXIF metadata.
+- [Markup agnostic queryable rich text with Portable Text](https://www.sanity.io/guides/introduction-to-portable-text).
```
[the remark this answers is not in the document]
Comment (kmelve) [reply README.md]: These are from our website - would it mean that it's hard to unpack there as well?

[the remark this answers is not in the document]
Comment (evenwestvang) [reply README.md]: I was thinking about the phrase `Markup agnostic queryable rich text with Portable Text` – I think that's unique to this context? Sry about lack of specificity – thought I hit the right line.

File: README.md (hunk 1 above)
Comment (evenwestvang) [reply README.md]: I was thinking about just linking to plugins? Again – sry about lack of specificity.

Comment (kmelve) [reply README.md]: Ah, directly from the ToC?

**Comment by kmelve** [comment]
Want to take a second look before we merge @evenwestvang?

**Pull request merged**